### PR TITLE
Also close the window on ctrl+w ctrl+c as expected

### LINKED ISSUE #483: <c-w><c-c> not working.
According to neovim docs,

```
|CTRL-W_CTRL-C|	CTRL-W CTRL-C	   same as "CTRL-W c"
```

so it should also close the window, but currently doesn't. It's a good thing to have because I can press `wcwcwc` a lot faster while my pinky holds `ctrl`. Pressing `ctrl+w c ctrl+w c ctrl+w c` is slower and error prone (easy to accidentally do `<c-w><c-c>`, which is probably why that map exists in neovim).

### PR DESCRIPTION
**Pull request by trusktr**
Fixes #483

### CODE REVIEW

File: vim/vscode-window-commands.vim [hunk 1]
```diff
@@ -59,6 +59,8 @@ nnoremap <C-w>q <Cmd>call VSCodeNotify('workbench.action.closeActiveEditor')<CR>
 xnoremap <C-w>q <Cmd>call VSCodeNotify('workbench.action.closeActiveEditor')<CR>
 nnoremap <C-w>c <Cmd>call VSCodeNotify('workbench.action.closeActiveEditor')<CR>
 xnoremap <C-w>c <Cmd>call VSCodeNotify('workbench.action.closeActiveEditor')<CR>
+nnoremap <C-w><C-c> <Cmd>call VSCodeNotify('workbench.action.closeActiveEditor')<CR>
+xnoremap <C-w><C-c> <Cmd>call VSCodeNotify('workbench.action.closeActiveEditor')<CR>
```
Comment (trusktr) [inline vim/vscode-window-commands.vim:63]: Do you prefer `<C-w><C-c>` capitalization format instead of `<c-w><c-c>` with all lowercase?

**Comment by asvetliakov** [comment]
Pulling latest master must fix test failures

File: vim/vscode-window-commands.vim (hunk 1 above)
Comment (samvv) [reply vim/vscode-window-commands.vim:63]: Looking at the other code, `<C-w><C-c>` seems like the way to go.

**Comment by samvv** [comment]
I rebased your branch on `master`. It seems like the tests were just canceled. Let's see if they pass with the latest version.

**Comment by samvv** [comment]
LGTM Unless @asvetliakov has something against this, I'll merge this in tomorrow.

**Review by asvetliakov — APPROVED** [review]

**Pull request merged**